.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2.1
+orbs:
```

Comment:
Most of our repos seem to be using docker images for their CI runs, see: https://github.com/Financial-Times/next-article/blob/master/.circleci/config.yml#L4

I'm not sure of the reasons behind using that (Kara might know more!) but would be best to be consistent with the rest of the FT.com estate if possible.

Reply:
i think the only reasoning behind Docker was that we migrated to CircleCI 2 before Orbs were available. however, it's good to keep things consistent